src/live_restore/live_restore_fs.c

```diff
@@ -1111,13 +1111,27 @@ __live_restore_fs_terminate(WT_FILE_SYSTEM *fs, WT_SESSION *wt_session)
     return (0);
 }
 
+/*
+ * __validate_live_restore_path --
+ *     Confirm that the given source directory is able to be opened.
+ */
+static int
+__validate_live_restore_path(WT_FILE_SYSTEM *fs, WT_SESSION_IMPL *session, const char *path)
+{
+    WT_FILE_HANDLE *fh;
+    /* Open the source directory. At this stage we do not validate what files it contains. */
+    WT_RET(
+      fs->fs_open_file(fs, (WT_SESSION *)session, path, WT_FS_OPEN_FILE_TYPE_DIRECTORY, 0, &fh));
```

Comment:
Per the tech design, should this also check for the existence of `WiredTiger.backup`?

Reply:
We could but there's an open question about whether we want to restrict ourselves to only starting on directories with .backup. In theory there is no reason we need that restriction.

Reply:
Ok. I just happened to review this right after reading that bullet about WiredTiger.backup in the Tech design.